camelot/core/files/storage.py

```diff
@@ -276,27 +280,28 @@ def delete(self, name: PurePosixPath):
         Path(self.path(name)).unlink(missing_ok=True)
 
 
+def get_hashed_path(path: PurePosixPath) -> str:
```

Comment:
maak hier aub een classmethod of een static methed op de HashStorage van, zodat de code netjes bij elkaar blijft